sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -1002,15 +1002,19 @@ private[hive] class HiveClientImpl(
 private[hive] object HiveClientImpl extends Logging {
   /** Converts the native StructField to Hive's FieldSchema. */
   def toHiveColumn(c: StructField): FieldSchema = {
+    val field = c.dataType match {
+      case TimestampNTZType => c.copy(dataType = TimestampType)
```

Comment:
I find this line confusing. Could you add comments here?

Reply:
Store timestamp_ntz in a hive-compatible way, by storing both ntz and ltz as hive timestamp